### PR TITLE
Feat transitions transaction size limit

### PR DESCRIPTION
**Pull request by antonydenyer**
All transactionSizeLimit to be specified in a transition:

```
"transitions" : [{
  "block": 200,
  "transactionSizeLimit": 128
}]
```

### CODE REVIEW

File: cmd/geth/config.go [hunk 1]
```diff
@@ -362,6 +367,13 @@ func quorumValidateConsensus(ethereum *eth.Ethereum, isRaft bool) {
 // the PrivacyEnhancements feature
 func quorumValidatePrivacyEnhancements(ethereum *eth.Ethereum) {
 	privacyEnhancementsBlock := ethereum.BlockChain().Config().PrivacyEnhancementsBlock
+
+	for _, transition := range ethereum.BlockChain().Config().Transitions {
+		if transition.PrivacyPrecompileEnabled {
+			privacyEnhancementsBlock = transition.Block
+		}
+	}
```
Comment (baptiste-b-pegasys) [inline cmd/geth/config.go]: I think we can avoid many ops as it is one way switch off to on
```suggestion
	if !privacyEnhancementsBlock {
		for _, transition := range ethereum.BlockChain().Config().Transitions {
			if transition.PrivacyPrecompileEnabled {
				privacyEnhancementsBlock = transition.Block
				break
			}
		}
	}
```

Comment (antonydenyer) [reply cmd/geth/config.go]: In theory it's not one way, you could decide to turn it off. Why you would do this I have node idea 😆

Comment (baptiste-b-pegasys) [reply cmd/geth/config.go]: but here, currently in the code you can't disable
not a pointer

Comment (antonydenyer) [reply cmd/geth/config.go]: 🤦 Good point!

**Pull request merged**